src/repository/comments.py

```diff
@@ -1,10 +1,29 @@
 from datetime import datetime
+from typing import List
 
+from sqlalchemy import Select, asc
 from sqlalchemy.orm import Session
 
 from src.database.models import User, PhotoComment
 from src.schemas import CommentSchema
 
+async def get_comment(comment_id:int, db:Session) -> PhotoComment | None:
```

Comment:
Do we need this function? No usages.

Reply:
It's a check used for next task. Just remove it for a while